tests/baselines/reference/moduleImport.errors.txt

```diff
@@ -1,13 +1,10 @@
-moduleImport.ts(2,17): error TS2339: Property 'Y' does not exist on type 'typeof X'.
 moduleImport.ts(2,17): error TS2694: Namespace 'X' has no exported member 'Y'.
 
 
-==== moduleImport.ts (2 errors) ====
+==== moduleImport.ts (1 errors) ====
     module A.B.C {
     	import XYZ = X.Y.Z;
     	               ~
-!!! error TS2339: Property 'Y' does not exist on type 'typeof X'.

```

Comment:
We drop this (duplicate-ish) error because now, rather than relying on `checkExpressionCached` to paint the reference recursively for the `import =` statement, we directly use `markIdentifierAliasReferenced` on the leftmost identifier in the entity name. Which should be faster, and, as this test shows, shouldn't cause us to drop any real errors, since any missing members get reported when we actually resolve the name (with a more sensible error, even).